### ISSUE TITLE
Shortcuts to move layer down/up, move frame left/right

### ISSUE THREAD

**Issue by c0ffeeartc** [aseprite/aseprite#837]
Can't find shortcuts to move layer down/up, move frame left/right.
Also search field in shortcut window would be handy, it's different issue though.

**Comment by dacap** [comment]
Added #849 about the search field in shortcut window

**Comment by cheeaun** [comment]
:+1: I can't seem to find a way to move layers up/down too :cry:

**Comment by dacap** [comment]
@cheeaun you can drag-and-drop layers (there is no shortcut but it's possible to move layers): http://www.aseprite.org/docs/move-layers/

**Comment by cheeaun** [comment]
@dacap ugh sorry, I should have read the docs  :disappointed_relieved: I was kind of dragging it right away without selecting it first (or I guess the first "drag" selected it and I didn't notice I can drag it again later  :neutral_face: ). Anyway, thanks!

**Comment by dacap** [comment]
@cheeaun you're welcome! :smile:

**Comment by dogboydog** [comment]
I agree this would be nice! Maybe even add an option under the Layer menu to Move Up and Move Down

**Comment by dacap** [comment]
I've just created a couple of scripts to do this:

https://gist.github.com/dacap/c70f3630a3cd24410efd81bd5a048171

**Comment by dogboydog** [comment]
Okay thanks a lot for the links. I got it set up with Alt + Up arrow or Alt + Down arrow and it's working great. 

I love Asesprite by the way! Great software

**Comment by dacap** [comment]
Glad to hear that! ^–^
